src/common/common/JsonDocLoader.py

```diff
@@ -13,6 +14,11 @@ def __init__(self, folder):
     def __iter__(self):
         for file_name in os.listdir(self.folder):
             file_path = os.path.join(self.folder, file_name)
-            file_content = open(file_path).read()
+            logging.info('JSON Pickling: ' + file_path)
 
-            yield jsonpickle.decode(file_content)
+            try:
+                file_content = open(file_path).read()
+                yield jsonpickle.decode(file_content)
+            except Exception as exception:  # pylint: disable=broad-except
+                logging.error("Unable to JSON pickle file: " + file_path, exception)
+                continue
```

Comment:
On exception, we should use logging.exception instead of (or in addition) logging.error, this way there is more info about the stacktrace in log : http://stackoverflow.com/questions/5191830/best-way-to-log-a-python-exception